lib/jobs/condition.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = conditionJobFactory;
+var di = require('di');
+
+di.annotate(conditionJobFactory, new di.Provide('Job.Evaluate.Condition'));
+di.annotate(conditionJobFactory, new di.Inject(
+    'Job.Base',
+    'Util',
+    'Logger'
+    )
+);
+
+function conditionJobFactory(
+        BaseJob,
+        util,
+        Logger
+) {
+    var logger = Logger.initialize(conditionJobFactory);
+
+     function ConditionJob(options, context, taskId) {
+        ConditionJob.super_.call(this, logger, options, context, taskId);
+    }
+    util.inherits(ConditionJob, BaseJob);
+
+    /**
+     * @memberOf ConditionJob
+     */
+    ConditionJob.prototype._run = function run() {
+        var when = this.options.when.toLowerCase();
```

Comment:
Needs an assertion for `options.when` or need a default handling routine if `options.when` is not defined.